pkgs/top-level/all-packages.nix

```diff
@@ -10316,6 +10316,8 @@ in
 
   cmucl_binary = pkgsi686Linux.callPackage ../development/compilers/cmucl/binary.nix { };
 
+  comby = callPackage ../development/tools/comby.nix { };
```

Comment:
This should get its own subdirectory, i. e. `../development/toosl/comby/default.nix`.

Reply:
```suggestion
  comby = callPackage ../development/tools/comby { };
```